models/org.go

```diff
@@ -401,8 +414,9 @@ func ChangeOrgUserStatus(orgID, uid int64, public bool) error {
 
 // AddOrgUser adds new user to given organization.
 func AddOrgUser(orgID, uid int64) error {
```

Comment:
This function should be removed since we have dropped the UI to add user to org.

Reply:
But user is still added to org when adding it to team

Reply:
That should be in a transaction but not this function

Reply:
true 👍

Reply:
I will address this in a separate PR